scalability_and_performance/telco-hub-rds.adoc

```diff
@@ -367,6 +374,13 @@ include::https://raw.githubusercontent.com/openshift-kni/telco-reference/release
 include::https://raw.githubusercontent.com/openshift-kni/telco-reference/release-4.19/telco-hub/configuration/reference-crs/optional/odf-internal/odfOperatorGroup.yaml[]
 ----
 
+[id="telco-hub-odfReady-yaml_{context}"]
+.odfReady.yaml
```

Comment:
@imiller0 should we also include references to this? it is just a policy to sync on ArgoCD deployment.